index.html

```diff
@@ -38,5 +40,6 @@
 		<div id="preact-app"></div>
 		<div id="vanillajs"></div>
 		<div id="inferno-app"></div>
+		<div id="ember-app"></div>
```

Comment:
I tried to get Ember to put its content into this div, but failed to. Not sure what I'm doing wrong. See https://github.com/CanopyTax/single-spa-examples/pull/37/files#diff-a9305e9d40f22872690636cbcfbd25b1R8